CHANGELOG.md

```diff
@@ -2,15 +2,32 @@
 
 ### Bugfixes
 
-* Lorem ipsum.
+* Fixed a few bugs (caused crashes) in pthread library on UWP.
+* Fixes a bug in chuncked binary column returning null value. 
+  Issue [#2418](https://github.com/realm/realm-core/pull/2418).
+* Possibly fixed some cases of extreme file size growth, by preventing starvation
+  when trying to start a write transaction, while simultaneously pinning an older
+  version. [#2395](https://github.com/realm/realm-core/pull/2395)
+* Fixed a bug when deleting a column used in a query [#2408](https://github.com/realm/realm-core/pull/2408)
+* Fixed a crash that occurred if you tried to override a binary with a size close
+  to the limit.
+* `seekpos()` and `seekoff()` in `realm::util::MemoryInputStreambuf` now behave
+  correctly when argument is out of range.
 
 ### Breaking changes
 
-* Lorem ipsum.
+* The table macros, supporting the typed interface, has been removed
+* Layout and version change for the .lock file required in order to prevent
+  starvation when waiting to start a write transaction (see above).
 
 ### Enhancements
 
-* Lorem ipsum.
+* Now supports case insensitive queries for UWP.
+* Upgraded Visual Studio project to version 2017.
+* Support handover of TableViews and Queries based on SubTables
+* Enable reading and writing of big blobs via Table interface.
+  Only to be used by Sync. The old interface still has a check on 
```

Comment:
@bmunkholm the line above. There is no "*" at the start of the line, so it is a continuation of the line above.